internal/handlers/common/aggregations/stages/collstats.go

```diff
@@ -29,12 +31,29 @@ import (
 	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
+// CollStatsDataSource fetches collection statistics from the database.
+type CollStatsDataSource interface {
```

Comment:
Let's call it `DataSource`. It has only `CollStats` for now, it will have `Query` or something soon and will be replaced with a backend interface later

Reply:
Sounds good, updated this name.